src/oscar/templates/oscar/catalogue/detail.html

```diff
@@ -78,9 +78,19 @@ <h1>{{ product.get_title }}</h1>
 
             <hr/>
 
-            {% block product_basket_form %}
-                {% include "catalogue/partials/add_to_basket_form.html" %}
-            {% endblock %}
+            {% if not product.is_parent %}
+                {% block product_basket_form %}
+                    {% include "catalogue/partials/add_to_basket_form.html" %}
+                {% endblock %}
+            {% else %}
+                <b>{% trans 'Variants:' %}</b><br>
```

Comment:
I don't think "variants" is a word we should be using with the end user. Not sure what to use instead. I also think it depends on how a project is using variants. I suggest we get wider input on whether projects would benefit from this before applying this change.

Reply:
> I don't think "variants" is a word we should be using with the end user. Not sure what to use instead.

We mostly use term "variant" for child product, so I don't think we have to invent anything here.

> I also think it depends on how a project is using variants.

We can wrap this snippet into the template block, so that it can be easily customized. Also, users have opportunity to change translation of the string to necessary word/phrase.

Reply:
Would prefer to drop the `b` tags and use a `h2` instead, and also wrap this in a block.